### ISSUE TITLE
No gradient defined for operation '..' (op type: QueueDequeueUpToV2)

### ISSUE THREAD

**Issue by YC-YunyingChen** [tensorflow/tensorflow#12864]
When I tried to compute the gradient of the loss with 'grads = opt.compute_gradients(total_loss, update_gradient_vars)', there was an error said 'No gradient defined for operation '..' (op type: QueueDequeueUpToV2)'

**Comment by cy89** [comment]
We can't backpropagate through queues; @vrv suggests that you probably have a queue between loss and update. 

This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
